README.md

```diff
@@ -0,0 +1,49 @@
+# Pulp Packaging
+
+## Adding a nightly built package
+
+To add a new project to be built nightly, add a new entry under pulp_packages in `package_manifest.yaml`
+
+```yaml
+my-new-package:
+      git: "{git url}"
+```
+
+1. Create a directory for the spec `packages/my-new-package/`
+2. Download/create a spec file in the above folder 
+3. run ansible-playbook nightlies.yaml -l my-new-package
+
+Tito is configured to checkout this project and generate a source tarbal to be used with the spec file under 
+packages/project-name
+
+## Updating a Dependency
+
+To add a new dependency, start by adding an entry to `package_manifest.yml` in the appropriate section.
+```yaml
+my-new-package:
+  files:
+    - "my-new-package-{version}.tar.gz"
+    - "my-new-package.spec"
+    - "patch-1.patch"
+```
+
+1. Create a directory for the spec and source(s) under packages/
+1. Download the spec and source(s)
+1. `git annex addurl {url} -file my-new-package-{version}.tar.gz`, where `{url}` is the Source0 line 
+    provided by `spectool my-package.spec` and `{version}` is the upstream version number of the dependency
+1. Edit the spec setting the release to `1`
+1. Scratch build your package 
```

Comment:
In the future, is the plan to add scratch build testing? We do this on foreman-packaging and katello-packaging today against Koji so should be easy enough to share this.

Reply:
Yes, I just haven't set that up yet.  I had scratch building working with nightly, but converted it over to non-scratch.  Won't be too hard to add back in